proc_scripts/phasing.py

```diff
@@ -304,15 +304,19 @@ def hermitian_function_test(s, down_from_max=0.5, rel_shift=1.5,shift_points=120
         fig, (ax1,ax2,ax3) = plt.subplots(3,1)
         fl.next('mirror tests', fig=fig)
         def real_imag_mirror(forplot, ax):
-            l = fl.plot(forplot.real, alpha=0.5, ax=ax, label='real')
-            fl.plot(forplot.C.setaxis('t2', lambda x: -x), color=l[-1].get_color(),alpha=0.2,ax=ax)
+            l = fl.plot(forplot.real, alpha=0.5, ax=ax, label='real',
+                    human_units=False)
+            fl.plot(forplot.C.setaxis('t2', lambda x: -x), color=l[-1].get_color(),alpha=0.2,ax=ax,
+                    human_units=False)
             l=fl.plot(forplot.imag,alpha=0.5,ax=ax)
             fl.plot(-forplot.imag.C.setaxis('t2',lambda x: -x),color=l[-1].get_color(),
-                    alpha=0.2,ax=ax)
+                    alpha=0.2,ax=ax,
+                    human_units=False)
             residual = forplot['t2':(-max_shift,max_shift)]
             if ndshape(residual)['t2'] % 2 == 0:
                 residual = residual['t2',:-1]
-            fl.plot(abs(residual-residual['t2',::-1].runcopy(np.conj))*20, ax=ax, label='residual x 20')
+            fl.plot(abs(residual-residual['t2',::-1].runcopy(np.conj))*20, ax=ax, label='residual x 20',
```

Comment:
why times 20?

Reply:
to make it bigger, because it should be very very small

Reply:
"it" being the residual which is equal to forplot['t2':(-max_shift,max_shift)]? and 20 is just an arbitrary choice?

Reply:
yes